src/ray/common/ray_config_def.h

```diff
@@ -692,7 +692,7 @@ RAY_CONFIG(std::string, predefined_unit_instance_resources, "GPU")
 /// The scheduler will treat these custom resource types as unit_instance.
 /// Default custom_unit_instance_resources is empty.
 /// When set it to "FPGA", we will treat FPGA as unit_instance.
-RAY_CONFIG(std::string, custom_unit_instance_resources, "")
+RAY_CONFIG(std::string, custom_unit_instance_resources, "num_neuron_cores")
```

Comment:
should it just be `neuron` or `neuron_core` ?  also do we differentiate upper/lower case? @rkooo567

Reply:
Looks like we differentiate upper/lower cases now